code/modules/events/immovable_rod.dm

```diff
@@ -163,6 +229,9 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 			num_sentient_mobs_hit++
 			if(iscarbon(smeared_mob))
 				num_sentient_people_hit++
+			if(dnd_style_level_up)
+				transform = transform.Scale(1.005, 1.005)
```

Comment:
```suggestion
				transform = transform.Scale(1.01, 1.01)
```
😏

Reply:
I haven't tested it because I'm lazy as shit, however if it turns out that the **penetrating** rod needs more **girth**, we can arrange it in a follow up PR where we **enhance** its **length**.